README.md

```diff
@@ -193,3 +200,134 @@ For more information on running and tearing down k8s environments, follow this [
 
 Because some of our modules pass headers bigger than 4k we had to increase the default value of the proxy buffer size for nginx.
 We also enable the CORS header for the applications that need it through the Ingress Rule.
+
+# Docker for Desktop - Mac
+
+## Prerequisites
+
+| Component   | Recommended version | Getting Started Guide |
+| ------------|:-----------: | ----------------------   |
+| Docker for Desktop | 18.06.1-ce   | https://www.docker.com/products/docker-desktop |
+| Homebrew           | 1.7.6        | https://brew.sh/         |
+| Helm               | 2.8.2        | https://docs.helm.sh/using_helm/#quickstart-guide |
+
+## Setup
+
+After installing Docker for Desktop you must enable Kubernetes and we recommend that you increase the memory and cores used by Docker for Desktop.
+
+- Memory mimimum: 8 GB
+- Core minimum: 4 cores
+
+We also recommend that you use Homebrew to install the Helm prerequisite.
+
+If you have previously deployed the DBP to AWS you will need to change/verify the ```docker-for-desktop``` context is being used.
+
+```bash
+kubectl config current-context                 # Display the current context
+kubectl config use-context docker-for-desktop  # Set the default context
+```
+
+### Helm Tiller
+
+Initialize the Helm Tiller:
+```bash
+helm init
+```
+
+### K8s Cluster Namespace
```

Comment:
We might want to consider dropping namespaces when deploying locally?

Reply:
Maybe we could just have a note somewhere at the end that talks about namespaces as an option?  The simpler we can make things for newcomers the better.

What do others think?

Reply:
I like that approach.  I'll modify it.